Project_CA/FileChanges.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project_CA
+{
+    public class FileChanges : Functionality
```

Comment:
I don't understand why do you need this inheritance.

Reply:
Indeed, inheritance at this point is incorrect.